static/app/views/performance/queues/destinationSummary/messageSpanSamplesPanel.tsx

```diff
@@ -356,6 +438,57 @@ function ConsumerMetricsRibbon({
 
 const SAMPLE_HOVER_DEBOUNCE = 10;
 
+const STATUS_OPTIONS = [
+  {
+    value: '',
+    label: t('All'),
+  },
+  ...[
+    'ok',
+    'cancelled',
+    'unknown',
+    'unknown_error',
+    'invalid_argument',
+    'deadline_exceeded',
+    'not_found',
+    'already_exists',
+    'permission_denied',
+    'resource_exhausted',
+    'failed_precondition',
+    'aborted',
+    'out_of_range',
+    'unimplemented',
+    'internal_error',
+    'unavailable',
+    'data_loss',
+    'unauthenticated',
+  ].map(status => {
+    return {
+      value: status,
+      label: t('%s', status),
```

Comment:
🤔 This is interesting. I agree with the intuition to translate this, but is this translated in the trace view, too? If not, this translated value won't match what people see there, and it'd be confusing. Might be better not to translate